Cineaste/ViewController/Settings/SettingsViewController.swift

```diff
@@ -79,6 +77,18 @@ class SettingsViewController: UITableViewController {
 }
 
 extension SettingsViewController {
+    func reloadUsernameCell() {
+        UIView.performWithoutAnimation {
+            let tableViewOffset = tableView.contentOffset
+
+            guard let rowForUsername = settings.firstIndex(of: SettingItem.name) else { return }
+            let indexPath = IndexPath(row: rowForUsername, section: 0)
+            tableView.reloadRows(at: [indexPath], with: .none)
+
+            tableView.contentOffset = tableViewOffset
```

Comment:
Do we really need this? I tested in on an iPhone 5s Simulator and it looked fine without this line. Was there a reason why you added this?

Reply:
You're correct, good find. Thanks for having a closer look! I actually found the solution online and didn't check. My bad. 😅

I've pushed a new commit that removes these lines.